src/heuristics/cro.rs

```diff
@@ -1,17 +1,36 @@
 //! Chemical Reaction Optimization (CRO).
+//!
+//! # References
+//!
+//! \[1\] Albert Y. S. Lam and Victor O. K. Li. 2010.
+//! Chemical-Reaction-Inspired Metaheuristic for Optimization.
+//! IEEE Transactions on Evolutionary Computation 14, 3 (June 2010), 381–399.
+//! DOI:<https://doi.org/10/bktgqf>
+//!
+//! \[2\] Albert Y. S. Lam and Victor O. K. Li. 2012.
+//! Chemical Reaction Optimization: a tutorial.
+//! Memetic Comp. 4, 1 (March 2012), 3–17.
+//! DOI:<https://doi.org/10/ggbmwk>
+//!
+//! \[3\] Albert Y. S. Lam, Victor O. K. Li, and James J. Q. Yu. 2012.
+//! Real-Coded Chemical Reaction Optimization.
+//! IEEE Transactions on Evolutionary Computation 16, 3 (June 2012), 339–353.
+//! DOI:<https://doi.org/10/fm8zqt>
 
 use crate::{
-    component::ExecResult,
-    components::*,
-    conditions::*,
-    configuration::{Configuration, ConfigurationBuilder},
+    components::{
+        boundary, initialization, misc, mutation, recombination, selection, utils, Block,
+    },
+    conditions::{self, LessThanN, RandomChance},
+    configuration::ConfigurationBuilder,
     identifier::{Global, Identifier, A, B},
     lens::common::PopulationSizeLens,
     logging::Logger,
-    problems::{LimitedVectorProblem, SingleObjectiveProblem},
+    problems::LimitedVectorProblem,
+    Component, Condition, Configuration, ExecResult, SingleObjectiveProblem,
 };
 
-/// Parameters for [real_cro].
+/// Parameters for [`real_cro`].
```

Comment:
Why add the "`"?

Reply:
So the link is properly formatted in monospaced font.